src/psyclone/parse/algorithm.py

```diff
@@ -147,6 +154,7 @@ def __init__(self, api="", invoke_name="invoke", kernel_path="",
         self._api = api
 
         self._arg_name_to_module_name = {}
+        self._arg_type_defns = {}
```

Comment:
Please add a comment to say what this will hold.

Reply:
Added.